### ISSUE TITLE
Exported png doesn't seem valid

### ISSUE THREAD

**Issue by burvil** [nickjj/ansigenome#68]
Are there any requirements for what ansigenome expects when does a scan?  I ask because I ran the following:

ansigenome scan directory-here/
ansigenome export -o /tmp/out.png

where directory-here contains my yaml playbooks and other include files that those yaml files refer to. 

However, when I open /tmp/out.png, it just shows what seem to be a series of unconnected lines, laid out sequentially in a row.  When I zoom in, it doesn't show me much; I can't make out anything for each line.  I expected to see a graph of relationships, much like is shown in the README file on this github site.

I thought the reason I'm not seeing anything may have been because I have yaml files that are essentially just dictionary definitions or other files that are referred to, i.e. included, in the actual playbooks that have roles and hosts defined, i.e. that have a roles: and a hosts: section in the .yml file.  So I copied to a subdirectory, and ran a scan on that subdirectory, but that doesn't even find any roles. 

What do I need to do to see a graph of relationships like shown in the README?

**Comment by nickjj** [comment]
Do you have dependencies set in your `meta/main.yml` files for your roles?

**Comment by burvil** [comment]
No, in fact, I don’t even have a directory named ‘meta’ in the directory I’m scanning.  I do, however, have main.yml in a series of subdirectories, relating to different roles. In addition, there are multiple references in playbooks to other files as well (e.g. that are included).   The only subdirectory in the directory to scan that matches meta/main.yml is roles/coreos-bootstrap/meta/main.yml.

Assuming I need such a file, can you tell me what that should look like?

**Comment by nickjj** [comment]
Here's the documentation: https://docs.ansible.com/ansible/devel/user_guide/playbooks_reuse_roles.html#role-dependencies

Here's about 100 real examples:
https://github.com/debops/debops/tree/master/ansible/roles

Some of the above roles have dependencies, others do not. The graph you see in the README was based off this project, but at this point it's been a very long while since a new one was generated.
